depthai_bridge/include/depthai_bridge/BridgePublisher.hpp

```diff
@@ -26,6 +26,7 @@
     #include <boost/make_shared.hpp>
     #include <boost/shared_ptr.hpp>
 
+    #include "depthai_ros_msgs/CameraMetadata.h"
```

Comment:
Can you add this to `#ifdef IS_ROS2` part ?
`#include "depthai_ros_msgs/msg/CameraMetadata.hpp"`